docs/.vuepress/components/WrapLayoutLiveDemoDoc.vue

```diff
@@ -0,0 +1,344 @@
+<template>
+  <StackLayout>
+    <StackLayout class="demo-panel">
+      <Label class="demo-panel__title" text="Container Configuration"></Label>
+      <StackLayout class="form-group">
+        <StackLayout orientation="horizontal">
+          <Label text="Background Color: " class="editor-param" width="200"/>
+          <input type="color" v-model="containerBackground" id="wraplayout_backgroundColor_input">
```

Comment:
we can use textfield instead of html input, and please replace all inputs

Reply:
done